packages/vega-scale/src/palettes.js

```diff
@@ -62,19 +75,21 @@ export const continuous = {
   darkRed: '3434347036339e3c38cc4037e75d1eec8620eeab29f0ce32ffeb2c'
 };
 
+const toStream = (scheme) => scheme.map((x) => x.slice(1)).join('').toLowerCase();
 export const discrete = {
-  category10: '1f77b4ff7f0e2ca02cd627289467bd8c564be377c27f7f7fbcbd2217becf',
+  accent: toStream(schemeAccent),
+  category10: toStream(schemeCategory10),
```

Comment:
I think it's funny that we are basically reversing the color generation from https://github.com/d3/d3-scale-chromatic/blob/2c52792197299346b7bdb94322bb4dff8f554fea/src/categorical/category10.js#L3.

Reply:
Small warning here: these are currently hexadecimal colors, but the format might change in the future. I don’t think in practice we will change them, but I don’t want to commit to never changing them. And certainly we might introduce new color schemes in the future that using a different format (_e.g._, to use the wider p3 color space).

Reply:
> I think it's funny that we are basically reversing the color generation
this is an irony very much not lost on me :)

> Small warning 
That's a very good point, but my intuition is that's a bridge that can be crossed when it's encountered. What would be the right way to guard against this change? Including a note saying to look out for that? Include a check in toStream that throws if it encounters not hex characters?

Reply:
Yeah, we can cross that bridge when we come to it. 

For now, I wonder whether we can remove the `toStream` method and recognize if we have an array of colors in the palette and skip the `colors` method in schemes.js.

Reply:
ok that appears to be fine